frontend/hooks/use-user-entries.ts

```
@@ -86,6 +91,175 @@ export function useUserEntries(): UseUserEntriesResult {
     return unsubscribe;
   }, [user, refetch]);
 
+  // Mark entries as seen
+  const markEntriesAsSeen = useCallback((entryIds: string[]) => {
+    setUnseenEntryIds(prev => {
+      const updated = new Set(prev);
+      entryIds.forEach(id => updated.delete(id));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)